bigquery_etl/query_scheduling/dag.py

```diff
@@ -24,19 +41,88 @@ class InvalidDag(Exception):
     pass
 
 
+@attr.s(auto_attribs=True)
+class DagDefaultArgs:
+    """Representation of Airflow DAG default_args."""
+
+    owner: str = attr.ib()
+    email: List[str] = attr.ib([])
+    depends_on_past: bool = attr.ib(False)
+    start_date: Optional[str] = attr.ib(None)
+    retry_delay: str = attr.ib("30m")
+    email_on_failure: bool = attr.ib(True)
+    email_on_retry: bool = attr.ib(True)
+    retries: int = attr.ib(2)
+
+    @owner.validator
+    def validate_owner(self, attribute, value):
+        if not is_email(value):
+            raise ValueError(f"Invalid email for DAG owner: {value}.")
```

Comment:
Alternatively we could do this validation by defining schemas. I thought it might be nice to have everything defined and validated in one place.